lambda_src/drivers/process_https.py

```diff
@@ -204,7 +199,9 @@ def process_row(
 
             next_url = (
                 cursor_value
-                if cursor_value and isinstance(cursor_value, str) and cursor_value.startswith('https://')
+                if cursor_value
```

Comment:
ok for now but let's try to move towards a process where these sorts of stray changes don't happen in a commit. when making changes, it's not just about what the code looks like, but what the `git blame` looks like, as understanding the history of the code is a valuable component of how the code is.

Reply:
makes sense, will keep this in mind 👍